lib/all-badge-examples.js

```diff
@@ -130,11 +130,19 @@ const allBadgeExamples = [
       },
       {
         title: 'SonarQube Coverage',
-        previewUri: '/sonar/http/sonar.qatools.ru/ru.yandex.qatools.allure:allure-core/coverage.svg'
+        previewUri: '/sonar/http/sonar.petalslink.com/org.ow2.petals%3Apetals-se-ase/coverage.svg'
       },
       {
         title: 'SonarQube Tech Debt',
-        previewUri: '/sonar/http/sonar.qatools.ru/ru.yandex.qatools.allure:allure-core/tech_debt.svg'
+        previewUri: '/sonar/http/sonar.petalslink.com/org.ow2.petals%3Apetals-se-ase/tech_debt.svg'
+      },
+      {
+        title: 'SonarQube Coverage (depreciated API)',
```

Comment:
Let's call this **(older version)** or **(legacy version)** or something similar, which is more from the users' perspective.

It may also be more accurate. I'm not sure whether the old versions always get deprecated, are you?

Reply:
`legacy API`/`legacy version` sounds good to me